yrs/src/transaction.rs

```diff
@@ -504,29 +479,42 @@ impl<'a> Transaction<'a> {
     /// This step is performed automatically when a transaction is about to be dropped (its life
     /// scope comes to an end).
     pub fn commit(&mut self) {
+        if self.committed {
```

Comment:
Now transaction has a `committed` flag, which is used to prevent multiple commits over the same transaction.